docs/development/select-screen.md

```diff
@@ -77,7 +77,7 @@ Configured choices may have text labels drawn with them. These are populated by
 
 ### Choice states
 
-Choices can have states, see [states](states.md). 
```

Comment:
Might be less confusing to have the states separate and link both there, same if more things have states going forward

Reply:
Yeah. I only made an initial stab at the docs. We still have to update them properly.